ivre/utils.py

```diff
@@ -2319,6 +2319,52 @@ def _ntlm_challenge_extract(challenge):
     return value
 
 
+def _ntlm_authenticate_info(request):
+    """
+    Extract host information in an NTLM_AUTH message
+    """
+    if (len(request) < 52):
+        LOGGER.warning("NTLM message is too short (%d) but should be at least "
+                       "52 char long", len(request))
+        return None
+
+    value = []
+    offset, ln = struct.unpack('IH', request[32:36] + request[28:30])
+    if ln > 0:
+        value.append("domain:" + \
+            encode_b64(_extract_substr(request, offset, ln)).decode())
```

Comment:
```suggestion
        value.append("domain:%s" %
                     encode_b64(_extract_substr(request, offset, ln)).decode())
```
This is better than adding strings